TidepoolOnboardingKitUI/View Models/OnboardingViewModel.swift

```diff
@@ -21,32 +21,68 @@ class OnboardingViewModel: ObservableObject, CGMManagerCreateNotifying, CGMManag
 
     let onboardingProvider: OnboardingProvider
 
-    @Published var isWelcomeComplete: Bool
-    @Published var isTherapySettingsComplete: Bool
+    @Published var sectionProgression: OnboardingSectionProgression
     @Published var therapySettings: TherapySettings?
 
     private lazy var cancellables = Set<AnyCancellable>()
 
     init(onboarding: TidepoolOnboardingUI, onboardingProvider: OnboardingProvider) {
         self.onboardingProvider = onboardingProvider
 
-        self.isWelcomeComplete = onboarding.isWelcomeComplete
-        self.isTherapySettingsComplete = onboarding.isTherapySettingsComplete
+        self.sectionProgression = onboarding.sectionProgression
         self.therapySettings = onboarding.therapySettings
 
-        $isWelcomeComplete
+        $sectionProgression
             .dropFirst()
-            .sink { onboarding.isWelcomeComplete = $0 }
-            .store(in: &cancellables)
-        $isTherapySettingsComplete
-            .dropFirst()
-            .sink { onboarding.isTherapySettingsComplete = $0 }
+            .sink { onboarding.sectionProgression = $0 }
             .store(in: &cancellables)
         $therapySettings
             .dropFirst()
             .sink { onboarding.therapySettings = $0 }
             .store(in: &cancellables)
     }
+
+    func titleForSection(_ section: OnboardingSection) -> String {
```

Comment:
[nit] Consider putting these into an `extension` to `OnboardingSection`, e.g.

```swift
extension OnboardingSection {
    var title: String {
        switch self {
        ...
        }
    }
    var duration: TimeInterval {
        switch self {
        ...
        }
    }
}
```

Reply:
That is how I originally had it, but upon reflection I decided it was better in the view model. From my perspective, the view model maps from the model (data) to the view (UI). The `OnboardingSection` enumeration is "data" and the view model translates that data into a form the UI can consume, namely the title and duration string.

Going to leave as-is.